submissions/andrewklmn/task_js_dom/app.js

```diff
@@ -22,20 +22,34 @@ const state = {
   activeIndex: null,
 };
 
+const menuItemClickListener = (event)=>{
+  let elem = event.target;
+  state.activeIndex = elem.parentNode.id;
+  renderPage(state);
+  event.stopPropagation();
+};
+
 const renderMenu = (state)=>{
   let nav = document.querySelector('nav');
   let prevUl = document.querySelector('ul');
   let fragment = document.createDocumentFragment();
-  let ul = document.createElement("div");
+  let ul = document.createElement("ul");
+
   state.items.forEach((item,index)=>{
     let li = document.createElement("li");
-    li.innerHTML = item.menu;
+    let a = document.createElement("a");
     li.id = index;
+    a.innerHTML = item.menu;
+    a.className = "menu-item";
```

Comment:
Use `ClassList` API for manipulating classes [Read more](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList)